<!-- Launch File for running svo with ceres backend from bag-file -->
<launch>

  <node pkg="sci_ros" type="sci_node.py" name="sci_node">
    <param name="sub_image_0" value='/cam0/image_raw' />
    <param name="sub_image_1" value='/cam1/image_raw' />    
    <param name="pub_image_0" value='/image_enhance_0' />
    <param name="pub_image_1" value='/image_enhance_1' />
    <param name="model_path" value="/home/sunteng/catkin_ws/src/sci_ros/ckpt/weights_122.pt"/>
  </node>
  <!-- SVO node -->
  <!-- RVIZ args=" -d $(find sci_ros)/rviz/euroc.rviz" -->
  <node name="vis" pkg="rviz" type="rviz" />

</launch>
